packages/manager/modules/support/src/support.scss

```diff
@@ -1,7 +1,5 @@
-.support-section {
-  @import '~bootstrap/scss/_functions';
-  @import '~bootstrap/scss/_variables';
-  @import '~bootstrap/scss/_mixins';
-  @import '~bootstrap/scss/_utilities';
-  @import '~bootstrap/scss/_grid';
-}
+@import '~bootstrap/scss/_functions';
```

Comment:
The style should keep its scope, it could cause weird behaviour as we have bs3 and bs4 cohabitating

Reply:
Well, it doesn't really resolve that if we have both BS3 and BS4 in the same app. It adds only more priority on the styles. Ideally, this part should not be managed by the module, but by the app since it's related to a style framework (contrary to a custom style).